src/Hl7.Fhir.Specification.Shared.Tests/Source/TerminologyTests.cs

```diff
@@ -70,6 +70,29 @@ public async T.Task ExpansionOfWholeSystem()
             //Assert.Equal("http://hl7.org/fhir/ValueSet/issue-type?version=3.14", ((FhirUri)versionParam.Value).Value);
         }
 
+        [Fact]
+        public async T.Task TestExpandingVsWithUnknownSystem()
+        {
+
+            var expander = new ValueSetExpander(new ValueSetExpanderSettings { ValueSetSource = _resolverWithoutExpansions });
+            var vs = new ValueSet
```

Comment:
You can create a new, empty InMemoryResolver to know 100% that it will not be resolved!